### ISSUE TITLE
feat: Does it support deployment to Docker and monitor database alerts?

### ISSUE THREAD

**Issue by hymn-github** [apitable/apitable#1701]
Just in Title

**Comment by mr-kelly** [comment]
Yes, APITable natively supports deployment via Docker (using `docker-compose`). You can find the installation guide in our README.

Regarding database alerts: APITable itself does not provide built-in database monitoring/alerting. We recommend using standard DevOps monitoring tools (like Prometheus + Grafana or Datadog) to monitor the underlying MySQL/Redis instances.
